modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/ObjectEntryManagerImpl.java

```diff
@@ -106,6 +112,48 @@ public ObjectEntry addObjectEntry(
 				new ServiceContext()));
 	}
 
+	@Override
+	public ObjectEntry addObjectRelationshipMappingTableValues(
+			DTOConverterContext dtoConverterContext, long userId,
+			ObjectDefinition objectDefinition, String objectRelationshipName,
+			long primaryKey1, long primaryKey2)
+		throws Exception {
+
+		List<ObjectRelationship> objectRelationships =
+			_objectRelationshipService.getObjectRelationships(
```

Comment:
Would it be a good option to define a new method in the Service to get directly the relationship by using the already implemented method `objectRelationshipPersistence.findByODI1_N` ? (If I understand correctly, that method would return the same element that we are trying to recover)

Reply:
Yes, that would be a good option but I don't know how much power we have to touch the code of other teams. When I develop new features for the REST APIs, I restrict myself to the methods provided by the services as I don't know the decisions made to expose one method and not to expose other ones.
cc/ @javierdearcos

Reply:
I think it is worth dedicating some time to check which part of logic or methods would be valuable for us to have in the service layer and open a PR with a conversation with the Objects team to know what they think about it.

If it is something that is straightforward I usually touch the services. Although in this case is better to have it as an isolated PR and open it to the team responsible for the service, in this case, objects.

We can continue moving forward this PR and then create the PR for the Objects team that will allow us to simplify our implementation in the REST layer. Then we can create a final PR refactoring our logic to use the new methods